### PR TITLE
Autopark level 17-18 done

### PR DESCRIPTION
**Pull request by AlexandarSviridenko**
Autopark level 17-18 done

### CODE REVIEW

File: src/main/java/com/Autopark/dto/VehicleDto.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
+package com.Autopark.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.ToString;
+
+    @Getter
+    @Builder
+    @ToString
+    public class VehicleDto {
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/dto/VehicleDto.java]: remove empty distance

File: src/main/java/com/Autopark/dto/VehicleTypeDto.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.Autopark.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+
+    @Getter
+    @Builder
+    public class VehicleTypeDto {
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/dto/VehicleTypeDto.java]: remove empty distance

File: src/main/java/com/Autopark/servlets/ViewCarTypesServlet.java [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.Autopark.Main.initInterfaceToImplementation;
+
+@WebServlet("/viewTypes")
+public class ViewCarTypesServlet extends HttpServlet {
+    private DtoService vehicleTypeService;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/servlets/ViewCarTypesServlet.java:32]: please, use @Autowired functionality which you create

File: src/main/java/com/Autopark/servlets/ViewCarsServlet.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@WebServlet("/viewCars")
+public class ViewCarsServlet extends HttpServlet {
+
+    private DtoService vehicleTypeService;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/servlets/ViewCarsServlet.java:28]: please, use @Autowired functionality which you create

File: src/main/java/com/Autopark/servlets/ViewDiagnosticServlet.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@WebServlet("/viewDiagnostic")
+public class ViewDiagnosticServlet extends HttpServlet {
+
+    private DtoService vehicleTypeService;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/servlets/ViewDiagnosticServlet.java:29]: please, use @Autowired functionality which you create

File: src/main/java/com/Autopark/servlets/ViewInfoServlet.java [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+@WebServlet("/info")
+public class ViewInfoServlet extends HttpServlet {
+
+    private DtoService vehicleTypeService;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/servlets/ViewInfoServlet.java:29]: please, use @Autowired functionality which you create

File: src/main/java/com/Autopark/servlets/ViewPlannedDiagnosticServlet.java [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@WebServlet("/viewPlannedDiagnostic")
+public class ViewPlannedDiagnosticServlet extends HttpServlet {
+
+    private DtoService vehicleTypeService;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/servlets/ViewPlannedDiagnosticServlet.java:29]: please, use @Autowired functionality which you create

File: src/main/java/com/Autopark/servlets/ViewReportServlet.java [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+@WebServlet("/viewReport")
+public class ViewReportServlet extends HttpServlet {
+
+    private DtoService vehicleTypeService;
```
Comment (AlexeyKatomachin) [inline src/main/java/com/Autopark/servlets/ViewReportServlet.java:28]: please, use @Autowired functionality which you create

File: src/main/java/com/Autopark/dto/VehicleDto.java (hunk 1 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/dto/VehicleDto.java]: fixed

File: src/main/java/com/Autopark/dto/VehicleTypeDto.java (hunk 2 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/dto/VehicleTypeDto.java]: fixed

File: src/main/java/com/Autopark/servlets/ViewCarTypesServlet.java (hunk 3 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/servlets/ViewCarTypesServlet.java:32]: fixed

File: src/main/java/com/Autopark/servlets/ViewCarsServlet.java (hunk 4 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/servlets/ViewCarsServlet.java:28]: fixed

File: src/main/java/com/Autopark/servlets/ViewDiagnosticServlet.java (hunk 5 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/servlets/ViewDiagnosticServlet.java:29]: fixed

File: src/main/java/com/Autopark/servlets/ViewInfoServlet.java (hunk 6 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/servlets/ViewInfoServlet.java:29]: fixed

File: src/main/java/com/Autopark/servlets/ViewPlannedDiagnosticServlet.java (hunk 7 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/servlets/ViewPlannedDiagnosticServlet.java:29]: fixed

File: src/main/java/com/Autopark/servlets/ViewReportServlet.java (hunk 8 above)
Comment (AlexandarSviridenko) [reply src/main/java/com/Autopark/servlets/ViewReportServlet.java:28]: fixed

**Review by nikoteressi — APPROVED** [review]
You should stop using Russian words in your code

**Comment by AlexandarSviridenko** [comment]
> You should stop using Russian words in your code

I needed using Russian words in JSP, this was write in Task.

**Pull request merged**